code-challenges.test.js

```diff
@@ -14,40 +17,82 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const object1 = { number: 15 }
+// Psuedocoding
+// create a test that returns whether a number is divisible by 3
+// input: number
+// output: number is divisble by 3
+describe("divisibleByThree", () => {
+    it("returns is divisible by three", () => {
+        let object1 = 15
+        expect(divisibleByThree(object1 % 3 === 0)).toEqual('${number} is divisible by three')
+        
+    })
+})
+// yarn test : ./code-challenges.test.js
+//   divisibleByThree
+//   ✕ returns is divisible by three
+
+// ● divisibleByThree › returns is divisible by three
+
+const divisibleByThree = (divisible) => {
+    if (object1 % 3 === 0){
```

Comment:
I see that you made the parameter 'divisible' but inside of your conditional you passed in 'object1' which is the provided argument. Be sure to keep consistent with your parameters so things don't get mixed up. If you need a better explanation I can cover this with you more in depth.